src/appleseed/renderer/modeling/light/sunlight.cpp

```diff
@@ -428,6 +482,15 @@ namespace
                 21097.9f, 20728.3f, 20240.4f
             };
 
+            // Limb darkening.
+            constexpr float LD_COEFICIENT = 0.6f; // Limb darkening coefficient for the sun for visible sunlight.
+            float limb_darkenig = 1.0f;
+            if (distance_to_center > 0.0f)
+                limb_darkenig = (1.0f - LD_COEFICIENT *
+                    (1.0f - std::sqrt(1.0f - std::pow(distance_to_center, 2.0f) /
```

Comment:
`std::pow(distance_to_center, 2.0f)` is just `square(distance_to_center)`, which is considerably faster to compute.

Would directly passing distance_to_center^2 make sense?

Reply:
does it make any difference?